kaggle_environments/envs/rps/utils.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+
+def get_score(left_move, right_move):
```

Comment:
Should this depend on `env.configuration.signs`?

Reply:
Interestingly I don't think so, the modulo step handles "wrapping" around the edge of the signs length from what I understand, but this is just a direct port of Myles' work so he'd likely understand better.